index.html

```diff
@@ -0,0 +1,74 @@
+</head>
+<body>
+    <!-- Logo -->
+    <header>
+        <div class="header">
+            <img src="images/logo.png" alt="Intuit Logo">
+        </div>
+    </header>
+    <!-- Main container-->
+    <main>
+        <!-- Form container -->
+        <div class="container">
+            <!-- Form heading -->
+            <div class="form-header text-center">
+                <h1>Create an Intuit account</h1>
+                <span class="sub-title">
+                    One account for everything Intuit, including Mint. <a href="https://accounts-help.lc.intuit.com/questions/1582580-creating-an-account">
+                        Learn more</a><br>
+                    Already have an account? <a href=#>Sign In</a>
+                </span>
+            </div>
+            <!-- Form -->
+            <form>
+                <fieldset class="form-main">
+                    <label for="email">Email address</label>
+                    <input type="email" name="email" id="email">
+
+                    <label for="phone">Phone <span class="font-300">(recommended)</span></label>
+                    <input type="tel" name="phone" id="phone">
+
+                    <label for="password">Password</label>
+                    <input type="password" name="password" id="password">
+
+                    <label for="confirm-password">Confirm Password</label>
+                    <input type="password" name="confirm-password" id="confirm-password">
+                </fieldset>
+                <!-- Submit button -->
+                <div class="submit-btn">
+                    <button type="submit" >
+                        <!-- <span class="lock-icon"></span> -->
+                        <img src="images/lock.svg" class="lock-icon" alt="Lock icon" />
+                        Create Account
+                    </button>
+                </div>
+            </form>
+            <!-- Agreement -->
+            <div class="agree-text text-center">
+                <p>By clicking Create account, you agree to our<br>
+                <a href="https://www.mint.com/terms">Terms</a> and have read and acknowledge our <a href="https://security.intuit.com/privacy">Privacy<br> Statement</a>.</p>
+            </div>
+            <div class="form-footer text-center">
+                <span>
+                    Invisible reCAPTCHA by Google <a href="https://www.google.com/intl/en/policies/privacy/">Privacy Policy</a> and <a href="https://www.google.com/intl/en/policies/terms/">Terms of Use.</a>
+                </span>
+            </div>
+        </div>
+    </main>
+    <!-- Footer -->
+    <footer>
+        <div class="footer text-center">
+            © 2020 Intuit, Inc. 
+            All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.<br>
+            Terms and conditions, features, support, pricing, and service options subject to change without notice.
+        </div>
```

Comment:
- [ ] Kindly add the three words above the disclaimer. Reminder: They are links!

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83826625-f3e46b00-a6a1-11ea-9648-02ed3409b0f0.png) | ![image](https://user-images.githubusercontent.com/57414789/83826647-fc3ca600-a6a1-11ea-913e-170e256701fd.png)